content/cloud-docs/run/ui.mdx

```diff
@@ -63,6 +63,14 @@ If you would rather automatically apply plans that don't have errors, you can [e
 
 To help you review proposed changes, Terraform Cloud can automatically run [speculative plans][speculative plan] for pull requests or merge requests.
 
+## Upgrading Terraform using Speculative Plans
+
+Using the UI, you can start a new speculative plan using the current configuration version but any terraform version available to the organization. After clicking **Start New Run**, use the select list to choose **Plan only** as the run type. This run mode reveals a new option to choose the specific terraform version.
```

Comment:
Question -- I noticed you only added a section here on the Ui page that incorporates upgrading with speculative plans + empty apply. But I noticed that we do have an API attribute for empty apply now. Is this a process that folks could also do through the API? Is that recommended/something we expect folks to do? If so, should we also add a section about this on the API page? https://www.terraform.io/cloud-docs/run/api#advanced-use-cases

I'm just thinking that since we've added to the API, maybe we should provide folks with guidance about how to do this through that workflow :)

Reply:
@laurapacilio This can certainly be done through the API as well; the UI even uses the API to do it, under the hood. But I'm feeling iffy about adding it to the API run workflow page, which has a very different vibe from this one and doesn't enumerate the various options in quite the same way.

Reply:
This is a fair point, and maybe someday I'll rework the API page so that it does talk more about the options! Okay @brandonc -> I'm okay with leaving this out. But I wanted to bring it up just in case!